src/components/ReadMore.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, useTheme } from "@mui/material";
+import { Typography } from "elements";
+import { FunctionComponent, ReactNode, useState } from "react";
+
+interface ReadMoreProps {
+  readonly abbreviatedContent: ReactNode;
+  readonly expandedContent: ReactNode;
```

Comment:
What do you think about making the ReadMore component take in 3 props, 
`characterCount: number` = Determine how many characters to show before the readmore logic displays
`text: string` = the full content to display
`typographyOverrides: object` = overwrite the props for the MUI typography component this would ultimately render 

The benefit of doing it this way would be so we can then do the manipulation/logic inside of this component and keep the parent components a little cleaner

Reply:
I dont think we need to handle the 2 different sales individually though... the rest of the content is the same, if anything i think it would help to make them display the same amount of characters(round down to the word) for consistency since it would take similar amount of space.

Reply:
Yea i dont see it being something different than a typography component that would need a read more logic

Reply:
I think there's a lot of room for interpretation in our mocks as we've seen previously though